### PR TITLE
Update Italian (it) Essentials, Exploring, Use, Use cases, and Staking buckets

### PR DESCRIPTION
**Pull request by minimalsm**
## Description
Update Italian (it) 

- Essentials
- Exploring
- Use Ethereum
- Use cases
- Staking

### CODE REVIEW

File: src/content/translations/it/dao/index.md [hunk 1]
```diff
@@ -111,6 +121,7 @@ Ethereum è la base perfetta per le DAO per una serie di motivi:
 - [Che cos'è una DAO e a cosa serve?](https://daohaus.substack.com/p/-what-is-a-dao-and-what-is-it-for) – [DAOhaus](https://daohaus.club/)
 - [Come avviare una comunità digitale basata su DAO](https://daohaus.substack.com/p/four-and-a-half-steps-to-start-a) – [DAOhaus](https://daohaus.club/)
 - [Cos'è una DAO?](https://coinmarketcap.com/alexandria/article/what-is-a-dao) – [Coinmarketcap](https://coinmarketcap.com)
+- [What is Holographic Consensus?](https://medium.com/daostack/holographic-consensus-part-1-116a73ba1e1c) - [DAOstack](https://daostack.io/)
```
Comment (minimalsm) [inline src/content/translations/it/dao/index.md:124]: Should be translated.

Comment (minimalsm) [inline src/content/translations/it/dao/index.md:124]:
```suggestion
- [Cos'è il Consenso Olografico?](https://medium.com/daostack/holographic-consensus-part-1-116a73ba1e1c) - [DAOstack](https://daostack.io/)
```

File: src/content/translations/it/decentralized-identity/index.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+- [Sign In with Ethereum and Decentralized Identity with Ceramic, IDX, React, and 3ID Connect](https://www.youtube.com/watch?v=t9gWZYJxk7c) — _Tutorial YouTube sulla creazione di un sistema di gestione delle identità per creare, leggere e aggiornare il profilo di un utente usando il suo portafoglio di Ethereum, di Nader Dabit_
+- [BrightID - Decentralized Identity on Ethereum](https://www.youtube.com/watch?v=D3DbMFYGRoM) — _Episodio del podcast Bankless, discussione su BrightID, una soluzione di identità decentralizzata per Ethereum_
+- [The Off Chain Internet: Decentralized Identity & Verifiable Credentials](https://www.youtube.com/watch?v=EZ_Bb6j87mg) — Presentazione di EthDenver 2022 di Evin McMullen
+
+### Community {#communities}
+
+- [ERC-725 Alliance on GitHub](https://github.com/erc725alliance) — _Sostenitori dello standard ERC725 per la gestione dell'identità sulla blockchain di Ethereum_
+- [SpruceID Discord server](https://discord.com/invite/Sf9tSFzrnt) — _Community di appassionati e sviluppatori al lavoro sulle funzioni di login con Ethereum_
```
Comment (minimalsm) [inline src/content/translations/it/decentralized-identity/index.md:184]: Fixed this in Crowdin

File: src/content/translations/it/social-networks/index.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
+---
+title: Social network decentralizzati
+description: Una panoramica dei social network decentralizzati su Ethereum
+lang: it
+template: use-cases
+emoji: ":mega:"
+sidebar: true
+sidebarDepth: 2
+image: ../../assets/ethereum-learn.png
```
Comment (minimalsm) [inline src/content/translations/it/social-networks/index.md:9]: Image not showing

File: src/intl/it/page-wallets-find-wallet.json [hunk 4]
```diff
@@ -106,7 +107,7 @@
   "page-find-wallet-new-to-wallets": "Non conosci i portafogli? Ecco una panoramica per iniziare.",
   "page-find-wallet-new-to-wallets-link": "Portafogli di Ethereum",
   "page-find-wallet-not-all-features": "Ad oggi nessun portafoglio ha tutte queste caratteristiche",
-  "page-find-wallet-not-endorsements": "I portafogli elencati in questa pagina non sono approvazioni ufficiali e sono indicati solo a scopo informativo. Le loro descrizioni sono state fornite dalle aziende che li hanno creati. Aggiungiamo questi prodotti a questa pagina basandoci su criteri delle",
+  "page-find-wallet-not-endorsements": "Wallets listed on this page are not official endorsements, and are provided for informational purposes only. Their descriptions have been provided by the wallet projects themselves. We add products to this page based on criteria in our",
```
Comment (minimalsm) [inline src/intl/it/page-wallets-find-wallet.json:110]: Shouldn't be English

File: src/content/translations/it/decentralized-identity/index.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
+---
+title: Identità decentralizzata
+description: Cos'è l'identità decentralizzata e perché importa?
+lang: it
+template: use-cases
+emoji: ":id:"
+sidebar: true
+sidebarDepth: 2
+image: ../../assets/eth-gif-cat.png
```
Comment (minimalsm) [inline src/content/translations/it/decentralized-identity/index.md:9]:
```suggestion
image: ../../../../assets/eth-gif-cat.png
```

File: src/content/translations/it/social-networks/index.md (hunk 3 above)
Comment (minimalsm) [inline src/content/translations/it/social-networks/index.md:9]:
```suggestion
image: ../../../../assets/ethereum-learn.png
```

File: src/content/translations/it/smart-contracts/index.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
+---
+title: Smart contract
```
Comment (corwintines) [inline src/content/translations/it/smart-contracts/index.md:2]: @minimalsm is this supposed to be in developer docs?

**Review by corwintines — APPROVED** [review]

File: src/intl/it/page-wallets-find-wallet.json (hunk 4 above)
Comment (corwintines) [reply src/intl/it/page-wallets-find-wallet.json:110]: Do you want to change this back first?

**Pull request merged**